locales/nb-NO/src/studio.ts

```diff
@@ -389,13 +389,13 @@ export default removeUndefinedLocaleResources({
   'form.error.no-array-item-at-key':
     'Ingen element med `_key` <code>"{{key}}"</code> funnet i stien <code>{{path}}</code>',
   /** The title above the error call stack output related to the crash */
-  'form.error.unhandled-runtime-error.call-stack.title': undefined, // 'Call Stack:'
+  'form.error.unhandled-runtime-error.call-stack.title': 'Kallstakk:',
   /** The title above the error component stack provided by React's underlying ErrorBoundary component */
-  'form.error.unhandled-runtime-error.component-stack.title': undefined, // 'Component Stack:'
+  'form.error.unhandled-runtime-error.component-stack.title': 'Komponentstakk:',
```

Comment:
Prefer to just leave this as english

```suggestion
  'form.error.unhandled-runtime-error.component-stack.title': 'Component Stack:',
```